rust/lancedb/src/index/scalar.rs

```diff
@@ -28,3 +28,13 @@
 pub struct BTreeIndexBuilder {}
 
 impl BTreeIndexBuilder {}
+
+/// Builder for a full text search index
+///
+/// A full text search index is an index on a string column that allows for full text search
+#[derive(Debug, Clone, Default)]
+pub struct FtsIndexBuilder {}
+
+impl FtsIndexBuilder {}
```

Comment:
IIUC, this is where things like tokenizer and stemmer will eventually go?

Reply:
Yes